src/modules/user-segment/user-segment.controller.spec.ts

```diff
@@ -5,16 +5,151 @@ import { UserSegmentService } from './user-segment.service';
 describe.skip('UserSegmentController', () => {
   let controller: UserSegmentController;
 
+  const mockUserSegment = {
+    id: '123',
+    createdAt: 1000,
+    updatedAt: 1000,
+    name: 'mockUserSegment',
+    all: {
+      connect: {
+        id: 7,
+      },
+    },
+    byPhone: {
+      connect: {
+        id: 8,
+      },
+    },
+    byId: {
+      connect: {
+        id: 9,
+      },
+    },
+  };
+
+  const mockUserSegmentService = {
+    create: jest.fn((dto) => {
+      return {
+        id: 123,
+        createdAt: 1000,
+        updatedAt: 1000,
+        ...dto,
+      };
+    }),
+
+    findAll: jest.fn(() => {
```

Comment:
Both `findOne` and `findAll` here return the `mockUserSegment` object. It would be better to define separate return objects for both.